exonum/src/blockchain/tests.rs

```diff
@@ -44,29 +46,24 @@ impl Service for TestService {
     }
 
     fn tx_from_raw(&self, raw: RawTransaction) -> Result<Box<dyn Transaction>, MessageError> {
-        Ok(Box::new(Tx::from_raw(raw)?))
+        Ok(TestServiceTxs::tx_from_raw(raw)?.into())
     }
 }
 
 transactions! {
     TestServiceTxs {
-        const SERVICE_ID = TEST_SERVICE_ID;
         struct Tx {
             value: u64,
         }
     }
 }
 
 impl Transaction for Tx {
-    fn verify(&self) -> bool {
-        true
-    }
-
-    fn execute(&self, view: &mut Fork) -> ExecutionResult {
+    fn execute(&self, mut tc: TransactionContext) -> ExecutionResult {
```

Comment:
In current implementation of `Fork`, its not possible to move `Fork` (we cant create `fork` of `fork`), but probably we could change this behaviour.

I am not sure about changing this to a borrowing, because we create context specificly for this transaction, and this is good to give transaction full ownership of this temporary object.

But in this design I am concerned about something else, that borrowing fork of `TransactionContext` leads to the inability to get other information from `context`. But granular fork could help us with this.